Components/GroupModules/GroupModule_GroupPosition/Config.xml

```diff
@@ -296,7 +356,7 @@
 							
 			<CheckButton parentKey="Locked" inherits="TellMeWhen_CheckTemplate">
 				<Anchors>
-					<Anchor point="TOPLEFT" relativeKey="$parent.Level" relativePoint="BOTTOMLEFT" x="-6" y="-7"/>
+					<Anchor point="BOTTOMLEFT" relativeKey="$parent.SizeY" relativePoint="BOTTOMRIGHT" x="9" y="-6"/>
```

Comment:
was easier for me to align it to the slider to the left since they are aligned to the same bottom